Misc/NEWS

```diff
@@ -10,6 +10,8 @@ Release date: XXXX-XX-XX
 Core and Builtins
 -----------------
 
+- Issue #26617: Fix crash when GC runs during weakref callbacks.
```

Comment:
Ah, this is my fault! Sorry. I executed `git diff` and `git log`, but their didn't show any changes. I needed to use `git diff --staged` or `git status` instead.